modules/test_instrument/NDB_BVL_Instrument_testtest.class.inc

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Can this file go into the test directory instead of creating a new module, since it's not a module that will be used outside of testing?